### DISCUSSION TITLE
Hide menu bar and title bar

### DISCUSSION THREAD

**Discussion by sccsatwork** [hluk/CopyQ#1619] [Ideas]
is that possible to hide men bar and title bar?

**Comment by hluk** [comment]
Hide menu bar: https://copyq.readthedocs.io/en/latest/faq.html#how-to-hide-the-menu-bar-in-the-main-copyq-window

Hide window title: Probably possible only in a window manager configuration or the somewhere in the desktop environment. Alternatively, use "Show the tray menu" shortcut (`copyq menu` command) to show simple menu instead of main window.

**Reply by sccsatwork** [reply]
thank you for the useful information!! 👍🏻 

i find that i can use ahk script to hide window title bar. url pasted below for reference.
https://superuser.com/questions/497908/is-there-any-way-to-hide-the-title-bar-of-a-program-using-autohotkey-in-windows
